magento/adobe-stock-integration
# #720 : Save a licensed image, then delete, then open grid, 'locate' button is shown on preview

This PR will fix the button issue if licensed image is deleted from the Media gallery then Open in media gallery will hide and Save button will be visible after reopen the Adobe Stock Modal
### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#720: Save a licensed image, then delete, then open grid, 'locate' button is shown on preview

### Manual testing scenarios (*)

1. Expand the preview for a licensed image
2. Save the licensed image to the media gallery
3. Select the saved image and delete it from the media gallery
4. Open the adobe stock grid once more, the preview for the previously-saved image should already be open

### Expected Result

`Open in media gallery` button  will be hidden and `save` button will be visible
